tools/run_tests/artifacts/build_artifact_python.bat

```diff
@@ -18,6 +18,7 @@ set PATH=C:\%1;C:\%1\scripts;C:\msys64\mingw%2\bin;C:\tools\msys64\mingw%2\bin;%
 python -m pip install --upgrade six
 @rem some artifacts are broken for setuptools 38.5.0. See https://github.com/grpc/grpc/issues/14317
 python -m pip install --upgrade setuptools==38.2.4
+python -m pip install --upgrade cython
```

Comment:
Just curious. Was something broken without this change?

Reply:
Yes. Python 2.7 and 3.5 are using Cython `0.25.x` which does not understand async generator syntax. After upgrade, they are able to compile the Cython code in Python 3 mode and all those AsyncIO stuff into C++.